docker-compose.template

```diff
@@ -0,0 +1,94 @@
+version: '2'
+
+services:
+
+  gateway-config:
+    image: nebraltd/hm-config:e0c6f88
```

Comment:
Let's move all of the git tags to the env file too so that we never need to touch the docker-compose template when we upgrade versions.